src/os/bluestore/BlueStore.cc

```diff
@@ -12333,22 +12339,24 @@ void BlueStore::_zoned_cleaner_thread() {
   ceph_assert(!zoned_cleaner_started);
   zoned_cleaner_started = true;
   zoned_cleaner_cond.notify_all();
-  std::deque<uint64_t> zones_to_clean;
+  auto a = dynamic_cast<ZonedAllocator*>(shared_alloc.a);
```

Comment:
suggest to add here and in other similar "post-cast" locations.
ceph_assert(a); 
ceph_assert(f);

Reply:
Done.